src/index.js

```diff
@@ -25,11 +27,20 @@ $seachForm.addEventListener('submit', function(event) {
         .then(renderUsers);
 })
 
+$sortUsersBtn.addEventListener("click", sorting);
+
 function renderUsers(usersData) {
+    gitUsers = usersData;
```

Comment:
your function should only render users from parameters, assigning users to the variable should be separate function